classes/Http/Controller/DashboardController.php

```diff
@@ -35,20 +35,29 @@ public function showSpeakerProfile()
     /**
      * Check to see if the CfP for this app is still open
      *
-     * @param integer $currentTime
+     * @param integer|\DateTimeInterface $currentTime
      *
      * @return boolean
      */
     public function isCfpOpen($currentTime = null)
     {
-        if (!$currentTime) {
-            $currentTime = strtotime('now');
+        if (! $currentTime) {
+            $currentTime = new \Datetime();
         }
 
-        if ($currentTime < strtotime($this->app->config('application.enddate') . ' 11:59 PM')) {
-            return true;
+        if (! $currentTime instanceof \DateTimeInterface) {
+            $currentTime = new \DateTime('@' . $currentTime);
         }
 
-        return false;
+        $enddate = new \DateTime($this->app->config('application.enddate'));
+        if ($enddate->format('H:i:s') == '00:00:00') {
+            $enddate->add(new \DateInterval('PT23H59M'));
```

Comment:
That's right. But as it was the current behavior I didn't want to break it as I couldn't see whether that's a wanted feature or not. If that was a bug all the time it's more than easy to change it here :)